composer/utils/object_store/oci_object_store.py

```diff
@@ -137,6 +137,7 @@ def download_object(
         filename: Union[str, pathlib.Path],
         overwrite: bool = False,
         callback: Optional[Callable[[int, int], None]] = None,
+        min_part_size: int = 128000000,
         num_parts: int = 10,
```

Comment:
As an aside, it also bugs me we just set this at 10 and split the file like this. Doesn't give much opportunity for work stealing and it might be faster if we ensure the requested content matches the chunk_size of the object storage.

Reply:
agree. this is hacky and not super optimized. im hoping OCI can just upstream chunked download -- this really should be captured in an object store SDK. (Fsspec might solve this?)

If we don't migrate to a new soln, I'll definitely look at tweaking